cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/DatasetServiceClient.java

```diff
@@ -69,6 +70,7 @@
   private static final Gson GSON = new Gson();
   private static final Type SUMMARY_LIST_TYPE = new TypeToken<List<DatasetSpecificationSummary>>() { }.getType();
   private static final Type MODULE_META_LIST_TYPE = new TypeToken<List<DatasetModuleMeta>>() { }.getType();
+  private static final HttpRequestConfig REQUEST_CONFIG = new HttpRequestConfig(60000, 60000);
```

Comment:
+1 for configurable.  IMO we shouldn't make any assumptions about any environment, and doesn't hurt anything to have a configuration option